docs/dsp-tools-create.md

```diff
@@ -250,194 +251,201 @@ A resource can be assigned to a taxonomic node within its properties. So a resou
 title "La Traviata" would have the property/attribute "musical-genre" with the value "Grand opera". Within the DSP,
 each property or attribute has an assigned cardinality. Sometimes, a taxonomy allows that an object may belong to 
 different categories at the same time (e.g. an image which depicts several categories at the same time). In these cases,
-a cardinality &gt; 1 allows to add multiple attributes
-of the same time. See further below the description of the [cardinalities](#cardinalities)
+a cardinality &gt; 1 allows to add multiple attributes of the same time. See further below the description of the
+[cardinalities](#cardinalities).
 
 A node of the Taxonomy may have the following elements:
 
-- _name_: Name of the node. This should be unique for the given list. The name-element is _optional_ but highly
-  recommended].
-- _labels_: Language dependent labels in the form ```{ "<lang>": "<label>, ... }```. The labels-element is _mandatory_. 
+- _name_: Name of the node. This should be unique for the given list. The name-element is optional but highly
+  recommended.
+- _labels_: Language dependent labels in the form `{ "<lang>": "<label>", ... }`. The labels element is mandatory. 
 It needs to specify at least one language.
-- _comments_: Language dependent comments (optional) in the form ```{ "<lang>": "<comment>, ... }```. The comments-element 
-  is _optional_.
-- _nodes_: Array of subnodes. If you have a non-hierarchical taxonomy (i.e. a taxonomy with only 2 levels, the root 
-  level and another level), you don't have child nodes. Therefore the nodes element can be omitted in case of a flat
+- _comments_: Language dependent comments in the form `{ "<lang>": "<comment>", ... }`. The comments element 
+  is optional.
+- _nodes_: Array of sub-nodes. If you have a non-hierarchical taxonomy (i.e. a taxonomy with only 2 levels, the root 
+  level and another level), you don't have child nodes. Therefore, the nodes element can be omitted in case of a flat
   taxonomy.
-  
-Each list must have exactely one root node which has the same form bu denotes the list itself.
 
 Here is an example on how to build a taxonomic structure with the help of JSON:
```

Comment:
is it "with the help of", not rather "in" JSON?

Reply:
absolutely ;)